_layouts/default.html

```diff
@@ -28,6 +28,13 @@
 		})();
 	</script>
 	<!-- End Matomo Code -->
+
+	<!-- Social Meta Tags -->
+	<meta property="og:title" content="{{ page.title  | default: 'CampusPulse Reporting System Page' }}">
+	<meta property="og:description" content="{{ page.description | default: 'This is a page for the CampusPulse Reporting System' }}">
+	<!-- Meta tag for image if applicable -->
+	<!-- <meta property="og:image" content="{{ page.image | absolute_url }}" /> -->
+	<meta property="og:url" content="{{ page.url | absolute_url }}" />
```

Comment:
i think theres a jekyll plugin that can handle this for us and is supported on GH pages (i think its named something like jekyll-seo-tags or something, if you look at the homepage for github.com/classclock or one of my other jekyll based sites youll probably find it